dask_sql/physical/rel/logical/project.py

```diff
@@ -70,6 +73,14 @@ def convert(self, rel: "LogicalPlan", context: "dask_sql.Context") -> DataContai
 
         # Actually add the new columns
         if new_columns:
+            # If the Dask DataFrame is currently empty we must do a little more work and convert the python
+            # literal into a Series so that Dask will assign the columns AND values
+            if len(df.columns) == 0:
+                new_columns = {
+                    k: dd.from_pandas(pd.Series([v]), npartitions=1).clear_divisions()
+                    for (k, v) in new_columns.items()
+                }
+
             df = df.assign(**new_columns)
```

Comment:
This `len` check can be expensive in the case of complex queries and should possibly be avoided if possible. It might be worth looking into what calcite was returning for these type of expressions and evaluate if this is needed.

Reply:
Is the `len` check an almost instantaneous operation on the `columns` since that is local data that doesn't require a compute? At first I was using `df.empty()` which I understand could be expensive. There was a warning from that saying to use this `len(df.columns)` instead.

Reply:
Ah, I actually missed it. You're right that `df.columns` is essentially a free check. I think this approach might work in that case. 
I just checked the calcite branch and it looks like we always default to returning a dataframe with 1 row if there's no input table as such so we don't run into the same issue as here.

Reply:
What does that resulting "empty" dataframe look like? Dimension wise? Just a dummy column and single value? Something like ...

```
| dummy_col |
0
```

Reply:
Yeah,
|0|
0